src/main/java/com/daitangroup/App.java

```diff
@@ -0,0 +1,24 @@
+package com.daitangroup;
+
+import com.daitangroup.interfaces.Vehicle;
+import com.daitangroup.vehicleImplementations.VWSaveiroCross;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
+
+        System.out.println( "Hello World!" );
```

Comment:
What is this line doing?

Reply:
Sorry, it was created with the project and i forgot to delete.